Bottom: 4d7b61bc27aa0965dc6ed9e23d046a92972ff434
Top:    a74eaa2323f8bd6db3a7cc00b06f0a54f8d55864
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-27 02:22:29 +0100

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index b410862..cacf7c6 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1790,7 +1790,7 @@ ghid_draw_everything (BoxTypePtr drawn_area)
       gui->set_layer ("invisible", SL (INVISIBLE, 0), 0)) {
     if (PCB->ElementOn) {
       r_search (PCB->Data->name_tree[NAME_INDEX (PCB)], drawn_area, NULL, name_callback, &side);
-      dapi->draw_layer (&(PCB->Data->Layer[max_copper_layer + side]), NULL); //drawn_area);
+      dapi->draw_layer (&(PCB->Data->Layer[max_copper_layer + side]), drawn_area, NULL);
     }
     if (global_view_2d) {
       r_search (PCB->Data->pad_tree, drawn_area, NULL, pad_callback, &side);
